packages/api/src/controllers/room.ts

```diff
@@ -259,7 +259,7 @@ app.get("/:roomId/user/:participantId", authorizer({}), async (req, res) => {
   res.json({
     id: participant.identity,
     state: participant.state,
-    joinedAt: participant.joinedAt,
+    joinedAt: participant.joinedAt * 1000,
```

Comment:
Is this the only place where we return a timestamp from livekit?

Reply:
There's another one too, are you thinking it needs a helper function?